### DISCUSSION TITLE
Json Type property Parameterisation issue with graphQL

### DISCUSSION THREAD

**Discussion by HaiderShahid** [prisma/prisma#12694] [Q&A]
I have a schema with type Json in it . When I try to insert data without query variables works but when I try to parameterise it. it fails.
mutation updatePropertyReport(
     $mlsListingObject: Json,
     $mlsID: String){
  updatePropertyReport(
                  data:{
                    mlsListingObject: $mlsListingObject
                  },
                  where:{mlsID: $mlsID}
	){
        id
        mlsID
    mlsListingObject
  }
}
query variables
{
  "mlsID": "14691538",
  "mlsListingObject":"{\"_id\":\"14691538\"}"
}
The following works the above doesn't with
mutation updatePropertyReport(
     $mlsID: String){
  updatePropertyReport(
                  data:{
                    mlsListingObject: "{\"_id\":\"14691538\"}"
                  },
                  where:{mlsID: $mlsID}
	){
        id
        mlsID
    mlsListingObject
  }
}

**Comment by austincrim** [comment]
Hey @HaiderShahid 👋 ,

When you say the first code snippet "doesn't work", are you getting a specific error message you can share with me? 
Where does Prisma fit into your GraphQL setup? 
Can you share the resolver this is using?

**Reply by vladi-stevanovic** [reply]
👋 Hello @HaiderShahid  I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! 🙌

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
